ui/pages/Project/components/ProjectCollaborators.jsx

```diff
@@ -129,42 +128,31 @@ CollaboratorRow.propTypes = {
   update: PropTypes.func,
 }
 
-const getSortedCollabs = (project, isAnvil) => orderBy(
-  (project.collaborators || []).filter(col => col.isAnvil === isAnvil), [c => c.hasEditPermissions, c => c.email],
-  ['desc', 'asc'],
-)
-
-const ProjectCollaborators = React.memo(({ project, onSubmit, addCollaborator }) => {
-  const localCollabs = getSortedCollabs(project, false)
-  const anvilCollabs = getSortedCollabs(project, true)
+const ProjectCollaborators = React.memo(({ project, user, onSubmit, addCollaborator }) => {
+  const canEdit = project.canEdit && !user.isAnvil
   return [
-    localCollabs.map(
-      c => <CollaboratorRow key={c.username} collaborator={c} update={project.canEdit ? onSubmit : null} />,
+    ...(project.collaborators || []).map(
+      c => <CollaboratorRow key={c.username} collaborator={c} update={canEdit ? onSubmit : null} />,
     ),
-    ((project.canEdit && !project.workspaceName) ? (
+    (canEdit ? (
```

Comment:
How about adding a message to tell how to add collaborators for the Anvil projects?

Reply:
No, how things work in AnVIL can change at any time without us even knowing and our explanation would become out of date. If someone really wants us to add that feature, they can submit a request for it later and they can figure out how to write a sufficiently generic method that won't get outdated but will still say something useful

Reply:
Actually, having though about this for another minute, we can add an explanation that just says the access is controlled in the workspace, without getting into the specifics of how to go about editing them